include/xsimd/config/xsimd_config.hpp

```diff
@@ -210,6 +210,9 @@
 // Workaround for MSVC compiler
 #ifdef _MSC_VER
 
+// NB: MSVC does not define __SSEn__
+#define XSIMD_WITH_SSE4_2 1
+
```

Comment:
This is a bit rude to always enable SSE4_2 for MSVC.
Per my understanding, 64bit MSVC supports at least SSE2, (/arch:SSE2 is actually an invalid option), and looks there's no way to detect SSE3/4 at compiler time.

Reply:
Obviously this change causes windows-clang ci job failures.
Comments welcomed.